guide/ops/locations/_openstack.md

```diff
@@ -32,12 +80,16 @@ Do this by setting the desired values id as an option in the
 
 ### Floating IPs
 
-Configuration of floating IPs is as networks; specify the pools to use as another
-[template option](#custom-template-options):
+The `autoAssignFloatingIp` option means that a [floating ip](https://www.mirantis.com/blog/configuring-floating-ip-addresses-networking-openstack-public-private-clouds/)
```

Comment:
You mention this value and what it does earlier (comment: [See comment below about this value]) -- perhaps create a small section that discusses this and both link to it?